### DISCUSSION TITLE
Não consigo evoluir no desafio dio-lab-open-source

### DISCUSSION THREAD

**Discussion by Nira12Ti** [digitalinnovationone/dio-lab-open-source#40010] [Desafios DIO]
Boa tarde, pessoal!

Estava muito feliz por estar fazendo o desafio, mas devo com certeza ter feito algo de errado no git bash, simplesmente nao consigo mais encontrar nada que ja havia feito. Aparece mensagem de erro, falha, arquivo não encontrado.
Por favor, me ajudem!

Um mundo totalmente novo pra mim e estou cada dia mais empolgada e apaixonada pela ideia de aprender e atuar na mesma.
Grata!

**Comment by AlanSilvaX** [comment]
Boa noite talvez voce esteja procurando no local errado aconteceu isso comigo fiquei 1h futucando quando vi tava abrindo no local errada rsrsrs

**Comment by Nira12Ti** [comment]
Olá Alan!
Estou ainda aqui na agonia… Estava sim rsrs
Muito obrigada!!!

Em seg., 24 de jun. de 2024 às 20:24, Alan Silva ***@***.***>
escreveu:

**Comment by wagnerov2001** [comment]
o meu tbm nao ta indo .

[a comment by an automated account is not shown]

**Comment by AdaoElias** [comment]
Eu fiquei 3 dias travado num desafio. Passei para uam outra atividade daí percebi qual era o objetivo real do exercicio e consegui resolver. Não desista!
Tente mais um pouco

**Comment by Dani-dba** [comment]
Eu estou iniciando agora, me sinto meio perdida, parece que ainda não entendi o real objetivo do desafio. 
Fiz o fork e estou seguindo as instruções do video, muita coisa que está na aula não está mais disponivel no READ.ME da plataforma e eu estou me virando nos 30.
agora estou procurando o botão de sync, nas instruções dela diz que é pra sincronizar com o repositorio original, eu consegui, mas nao encontrei o sync nessa pagina, somente na pagina do meu fork. 
to seguindo o fluxo mas to bem perdida.....

[a comment by an automated account is not shown]

**Comment by Pessoapvs** [comment]
vou ter que refazer todo o processo o meu tambem não deu certo....

**Comment by wagnerov2001** [comment]
A questao e que o passo a passo nao funciona, acredito que houve
atualizacoea e nao foi feito um novo vidio .

Em dom, 22 de set de 2024 11:43, Person Vieira ***@***.***>
escreveu:

**Reply by Jeff1611** [reply]
Te entendo, você também não está conseguindo criar um FORK?

**Reply by santioliverag** [reply]
você conseguiu? Eu não posso fazer um fork

**Comment by Jeff1611** [comment]
Eu não estou conseguindo criar um Fork, aparece que: "Não há destinos disponíveis para bifurcar este repositório."
